CoreFoundation/Base.subproj/CoreFoundation.h

```diff
@@ -33,6 +35,7 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
+#include <netdb.h>
```

Comment:
Which head is which of the two? I can remove it from the Darwin version and add #if __has_include(). The symbols are used in NSHost.swift.